lua/telescope/config.lua

```diff
@@ -323,6 +323,33 @@ local telescope_defaults = {
                           Default: 1000
     ]],
   },
+  previewers = {
+    {
+      filesize_limit = 25,
+      timeout = 250,
```

Comment:
another thing we can do (maybe out of scope for this PR) is a treesitter field. Where we can then say `treesitter = false` to disable treesitter highlighting and `treesitter = { "lua", "c" }` to enable treesitter only for a subset of languages.

Would resolve https://github.com/nvim-telescope/telescope.nvim/issues/857

Reply:
Yeah, I'll look into this. I'd say we should do this as part of this PR.

Reply:
Added. I've also changed the configuration from `previewer` to `preview` as I kind of prefer to have these two disentangled. Let me know if you prefer `previewer` at setup though.

Reply:
i would prefer `previewer` just because we currently can already disable with `Telescope find_files previewer=false`. I dont know tho, because maybe i wanna do `:lua require("telescope.builtin").find_files { previewer = { timeout = 1000 } }` and that would override the previewer handle.

Maybe `preview` is fine. I dont wanna write this in stone yet, i need to think about it.